### ISSUE TITLE
App doesn't run on MacOs 10.13

### ISSUE THREAD

**Issue by courbeges** [SartoxSoftware/audacium#57]
Hi everybody,

   There seems to be a problem with the MacOs version 

https://github.com/SartoxSoftware/audacium/releases/download/v1.0.0/Audacium_macOS_Intel_1495198346_7c3a6d5.zip

The zip file can be unzipped without problem, but then the application (audacity.app) won't run (see the attached screenshots.
The app "audacity.app" there is really audacity (2.4.2), and audacium should be the app "audacity 2.app", which cannot be opened...)

<img width="1022" alt="audacium-macos-10-13" src="https://user-images.githubusercontent.com/67154879/143239175-48720cdd-06e3-47f8-8f0c-431dda695e1e.png">
<img width="436" alt="audacium-macos-10-13-b" src="https://user-images.githubusercontent.com/67154879/143239210-9b3b037f-507f-4a59-8186-4dacc5408140.png">

What am I missing here?
Thanks a lot in advance for any clue.

**Comment by AnErrupTion** [comment]
I don't own a Mac so sadly I couldn't test the binary before uploading. However I'm guessing it's just a packaging problem since there shouldn't be any problem with the code. Could you try to build Audacium yourself and try the built binary?

**Comment by AnErrupTion** [comment]
@courbeges A quick question, did you test opening Audacity.app at all?

**Comment by courbeges** [comment]
Yes I did. Here are more details:

* In fact, I compiled Audacium some time ago (in August) from sources on
Linux Ubuntu, and after some trials, it was ok (except some plugins crashing).

* I asked the question about this compilation problem on Mac for a friend of mine,
who tried to install it and didn't succeed. I have no access to his machine (due to
pandemic situation), and I don't know much about Mac system. On the other hand,
my friend is not an Apple guru, and never tried to compile a software from sources.

* Since then, I have sent him a script, that would possibly install everything and
compile the sources, but my friend had no time to try it.

* As of the version I compiled on Linux, it seems to be slightly different from the
appimage version (some plugins in particular do not run the same way on both versions).
I can provide details on demand.

All the best.

**Comment by courbeges** [comment]
And as for Audacity.app on my friend's Mac, it is the original version of Audacity (2.4.2 I guess), running ok.

**Comment by AnErrupTion** [comment]
I don't own a Mac but as soon as I can make a Mac VM, I'll try fixing that. Unfortunately, for the time being, I can't do much. Sorry about that.

**Comment by AnErrupTion** [comment]
Oh, a dumb question, but are you using an ARM-based or Intel-based Mac? @courbeges

**Comment by courbeges** [comment]
My friend's Mac is Intel based.

**Comment by AnErrupTion** [comment]
Currently, Audacium only works on Intel-based Macs.

**Comment by courbeges** [comment]
Ok. So it should work on my friend's Mac... :-\

**Comment by AnErrupTion** [comment]
@courbeges Any updates?

**Comment by courbeges** [comment]
Not really. I have tried to write a script and send it to my friend, but at some point something 
couldn't be installed (if I remember correctly, there was a problem with the version of 'cmake',
not handled by the version of the system). So we gave up, hoping for future versions...

**Comment by AnErrupTion** [comment]
That's weird. Although I think the main problem here is about packaging, more specifically that we aren't renamed everything yet. I'll be looking at that soon.

**Comment by badlydrawnface** [comment]
I have had the exact same error trying to build Audacium on an M1 MacBook Pro for x64 and Universal.

**Comment by AnErrupTion** [comment]
@badlydrawnface Audacium does not work on M1 Macs yet.
